### DISCUSSION TITLE
Indicate new comments and comments from other accounts

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#14820] [Pull Requests]
I'm looking at:
https://github.com/pulls?q=is%3Aopen+is%3Apr+author%3Ajsoref+archived%3Afalse+org%3Aprimer
<img width="979" alt="image" src="https://user-images.githubusercontent.com/2119212/163071311-31f54b96-59a4-499b-98f7-136ec0734952.png">

I can see there are some comments. But, on average, most of those comments are probably mine. I'd like a way to visibly determine that:
* there are new comments I haven't seen (sure, I could use the 🔔, but I have lots of things in there)
* there are comments from an account other than me

**Comment by suryanshmak** [comment]
you could easily determine that by looking at the commit history and if you clone the repository, GitLens does it for you.
